conn/node.go

```diff
@@ -640,34 +640,34 @@ var readIndexOk, readIndexTotal uint64
 
 // WaitLinearizableRead waits until a linearizable read can be performed.
 func (n *Node) WaitLinearizableRead(ctx context.Context) error {
-	span := otrace.FromContext(ctx)
-	span.Annotate(nil, "WaitLinearizableRead")
+	span := trace.SpanFromContext(ctx)
+	span.AddEvent("WaitLinearizableRead")
 
 	if num := atomic.AddUint64(&readIndexTotal, 1); num%1000 == 0 {
 		glog.V(2).Infof("ReadIndex Total: %d\n", num)
 	}
 	indexCh := make(chan uint64, 1)
 	select {
 	case n.requestCh <- linReadReq{indexCh: indexCh}:
-		span.Annotate(nil, "Pushed to requestCh")
+		span.AddEvent("Pushed to requestCh")
 	case <-ctx.Done():
-		span.Annotate(nil, "Context expired")
+		span.AddEvent("Context expired")
 		return ctx.Err()
 	}
 
 	select {
 	case index := <-indexCh:
-		span.Annotatef(nil, "Received index: %d", index)
+		span.AddEvent(fmt.Sprintf("Received index %d", index))
```

Comment:
will it make sense to create a helper function around this instead of using Sprintf everywhere?